demo/server/src/modules/index.ts

```diff
@@ -4,10 +4,13 @@ import * as express from 'express';
 import { connect as syncConnector } from './datasync/Connector';
 import { router as syncRouter } from './datasync/Router';
 import { init as authInit } from './passport-auth';
+import { init as initKeycloak } from './keycloak';
 
 export let securityMiddleware: EndpointSecurity;
 export let logger: Logger;
 
+const useKeycloak = true;
```

Comment:
[Optional/Idea] We can connect that with existence of the environment variable that points to keycloak server.